rules/Php55/Rector/String_/StringClassNameToClassConstantRector.php

```diff
@@ -85,7 +85,7 @@ public function run()
                 [
                     'ClassName',
                     'AnotherClassName',
-                    StringClassNameToClassConstantRector::SHOULD_KEEP_PRE_SLASH => false,
+                    self::SHOULD_KEEP_PRE_SLASH => false,
```

Comment:
It should not be changed to `self::` in code example

Reply:
it seems changed by ecs:

```diff
     public function __construct(
         private readonly ReflectionProvider $reflectionProvider,
     ) {
@@ -85,7 +85,7 @@
                 [
                     'ClassName',
                     'AnotherClassName',
-                    StringClassNameToClassConstantRector::SHOULD_KEEP_PRE_SLASH => false,
+                    self::SHOULD_KEEP_PRE_SLASH => false,
                 ],
             ),
         ]);
    ----------- end diff -----------


Applied checkers:

 * PhpCsFixer\Fixer\ArrayNotation\WhitespaceAfterCommaInArrayFixer
 * PhpCsFixer\Fixer\ClassNotation\ClassAttributesSeparationFixer
 * PhpCsFixer\Fixer\ClassNotation\OrderedClassElementsFixer
 * PhpCsFixer\Fixer\ClassNotation\SelfAccessorFixer
 * PhpCsFixer\Fixer\Operator\NotOperatorWithSuccessorSpaceFixer
 * PhpCsFixer\Fixer\Operator\UnaryOperatorSpacesFixer
 * Symplify\CodingStandard\Fixer\LineLength\LineLengthFixer
```

Reply:
it seems caused by `PhpCsFixer\Fixer\ClassNotation\SelfAccessorFixer` from php-cs-fixer rule, we may need to skip it in `ecs.php`

Reply:
I created PR :

- https://github.com/rectorphp/rector-src/pull/5363

for it.